cluster/manifests/kube-state-metrics/rbac.yaml

```diff
@@ -78,6 +78,7 @@ rules:
   - autoscaling
   resources:
   - horizontalpodautoscalers
+  - verticalpodautoscalers
```

Comment:
Does this work? I would imagine `autoscaling.k8s.io` is a different `apiGroup` than `autoscaling`?

Reply:
That's what it claimed it needs in the error message.

Reply:
Ah, no, `autoscaling.k8s.io` is not equal to `autoscaling` I guess :face_with_head_bandage:

Reply:
Hopefully fixed.